### ISSUE TITLE
Quadratic Weighted Kappa

### ISSUE THREAD

**Issue by drbeh** [Project-MONAI/MONAI#3164]
**Is your feature request related to a problem? Please describe.**
Quadratic Weighted Kappa (QWK) measures the similarity between the predictions and targets and is the main metric for PANDA challenge.

**Comment by myron** [comment]
So the QWK is just a one-line to compute in numpy.  I think we can just call it directly when needed instead of adding this as a Metric to monai. What do you think?

```
from sklearn.metrics import cohen_kappa_score
qwk = cohen_kappa_score(PREDS, TARGETS, weights='quadratic')

```

**Comment by drbeh** [comment]
@myron, sure, that makes sense. If you are not relying on a torch.Tensor computation and it works for you, let's keep it that way for now.
